src/styles/index.scss

```diff
@@ -1,3 +1,80 @@
-body {
+@keyframes revolution-second-hand {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes revolution-minute-hand {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__object {
+    position: relative;
+    height: 80vmin;
+    width: 80vmin;
+    border: 1vmin dotted #000;
```

Comment:
The color value `#000` can be replaced with the keyword `black` to improve code readability according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).